episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+:::::::::::::::::::: instructor
+
+We can showcase this last bias using the [concept described in this `{cfr}` vignette](https://epiverse-trace.github.io/cfr/articles/cfr.html#concept-how-reporting-delays-bias-cfr-estimates).
+
+<!-- create code and then a .gif? -->
+
+::::::::::::::::::::
+
+:::::::::::::::::::: testimonial
+
+Improving an _early_ epidemiological assessment of a delay-adjusted CFR is crucial for determining virulence, shaping the level and choices of public health intervention, and providing advice to the general public. 
+
+In 2009, during the swine-flu virus, Influenza A (H1N1), Mexico had an early biased estimation of the CFR. Initial reports from the government of Mexico suggested a virulent infection, whereas, in other countries, the same virus was perceived as mild ([TIME, 2009](https://content.time.com/time/health/article/0,8599,1894534,00.html)).
+
+In the USA and Canada, no deaths were attributed to the virus in the first ten days following the World Health Organization's declaration of a public health emergency. Even under similar circumstances at the early stage of the global pandemic, public health officials, policymakers and the general public want to know the virulence of an emerging infectious agent.
+
+[Fraser et al., 2009](https://www.science.org/doi/full/10.1126/science.1176062) reinterpreted the data assessing the biases and getting a clinical severity lower than the 1918 influenza pandemic but comparable with that seen in the 1957 pandemic.
+
+::::::::::::::::::::
+
+:::::::::::::::::::: discussion
+
+Based on your experience:
+
+- Share any previous outbreak in which you participated in its response.
+
+Answer to these questions:
+
+- How did you assess the clinical severity of the outbreak?
+- What were the primary sources of bias?
+- What did you do to take into account the identified bias?
+- What complementary analysis would you do to solve the bias?
+
+:::::::::::::::::::: 
+
+## Delay-adjusted CFR
+
+Nishiura et al., 2009 developed a method that considers the time delay from the onset of symptoms to death.
+
+Real-time outbreaks may have a number of deaths that are insufficient to determine the time distribution between onset and death. Therefore, we can estimate the _distribution delay_ from historical outbreaks or reuse the ones available in systematic reviews from repositories like `{epiparameter}` or `{epireview}`.
+
+Let's use `{epiparameter}`:
+
+```{r, message=FALSE, warning=FALSE}
+# Get delay distribution
+onset_to_death_ebola <-
+  epiparameter::epidist_db(
+    disease = "Ebola",
+    epi_dist = "onset_to_death",
+    single_epidist = TRUE
+  )
+```
+
+To calculate the delay-adjusted CFR, we can use the `cfr_static()` function with the `data` and `delay_density` arguments.
+
+```{r}
+# Calculate the static CFR while adjusting for delays
+cfr_static(
+  data = ebola1976,
+  delay_density = function(x) density(onset_to_death_ebola, x)
+)
+```
```

Comment:
The format in which the delay distribution is passed to the `delay_density` argument could be explained more - maybe with a visual example? E.g. a plot of the PDF over time. This can also help explain the convenience of using the epidist class.

Reply:
I added the `plot()` and a spoiler callout on the internal method, redirecting to `estimate_outcomes()`. Added in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/1ac361d2b3b7241cf43dc1c008225f534948b0a5

If a visual can make a better explanation, we can fill an issue and draft a visual and get support from Carmen to make this visually appealing.